proto/panacea/datadeal/v2alpha2/genesis.proto

```diff
@@ -15,4 +16,28 @@ message GenesisState {
   repeated DataSale data_sales = 3 [(gogoproto.nullable) = false];
   repeated DataVerificationVote data_verification_votes = 4 [(gogoproto.nullable) = false];
   repeated DataDeliveryVote data_delivery_votes = 5 [(gogoproto.nullable) = false];
+  repeated DataVerificationQueue data_verification_queue = 6 [(gogoproto.nullable) = false];
+  repeated DataDeliveryQueue data_delivery_queue = 7 [(gogoproto.nullable) = false];
+}
+
+// DataVerificationQueue defines a data verification queue.
+message DataVerificationQueue {
```

Comment:
I think the name `DataVerificationQueue` change would be good. It is not `Queue`, just the element of `Queue`. So, How about `DataVerificationQueueElement`?

Reply:
Yes, I think you're right. It is the element that consist of the queue. I'll change the name.

Reply:
Renamed in [e3286e0](https://github.com/medibloc/panacea-core/pull/496/commits/e3286e057d4158f89e04a401b51b03f212c2cc86).